client/lib/text-utils/index.js

```diff
@@ -24,4 +30,26 @@ function countWords( content ) {
 	return 0;
 }
 
-export default { countWords };
+const diffWords = JsDiff.diffWords;
+
+function countDiffWords( diffChanges ) {
+	return reduce(
+		diffChanges,
+		( accumulator, change ) => {
+			const count = countWords( change.value );
+			change.added && ( accumulator.added += count );
+			change.removed && ( accumulator.removed += count );
+			return accumulator;
+		},
+		{
+			added: 0,
+			removed: 0,
+		},
+	);
+}
+
+export default {
```

Comment:
Created a new PR to make sure we don't drop this: #14794

it's based on this one so we should merge this one first.